app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -39,23 +44,53 @@ def usps_proofer
     end
 
     def create_usps_enrollment(enrollment, pii)
+      address = pii['address1']
+      address += " #{pii['address2']}" unless pii['address2'].blank?
+
       applicant = UspsInPersonProofing::Applicant.new(
         {
           unique_id: enrollment.usps_unique_id,
           first_name: pii['first_name'],
           last_name: pii['last_name'],
-          address: pii['address1'],
-          # do we need address2?
+          address: address,
           city: pii['city'],
           state: pii['state'],
           zip_code: pii['zipcode'],
           email: 'no-reply@login.gov',
         },
       )
+
       proofer = usps_proofer
+      response = nil
+      begin
+        response = proofer.request_enroll(applicant)
+      rescue Faraday::BadRequestError => err
+        handle_bad_request_error(err, enrollment)
+      rescue StandardError => err
+        handle_standard_error(err, enrollment)
+      end
 
-      response = proofer.request_enroll(applicant)
-      response['enrollmentCode']
+      response&.enrollmentCode
```

Comment:
Based on the class implementation below, the `attr_reader` is in snake case. Should this be ... ?

```suggestion
      response&.enrollment_code
```

Reply:
Woops. Yep, fixed